e-commerce/payment-backend/src/operations.ts

```diff
@@ -170,15 +170,15 @@ export class PlaidPayments {
 
   @Communicator()
   static async paymentWebhook(
-    _ctxt: CommunicatorContext, 
+    ctxt: CommunicatorContext,
     webhook: string, 
     session_id: string, 
     status: string | undefined, 
     client_reference_id: string | undefined
   ): Promise<void>
   {
-    if (webhook === "http://fakehost/webhook") {
-        return; // In testing - but it would be good to confirm this by checking the environment
+    if (ctxt.getConfig('unittest', false) && webhook === "http://fakehost/webhook") {
+      return; // In testing, matching the bogus testing URL
```

Comment:
That part is interesting. I didn't realize we would have to modify production code for the sake of unit testing. Ideally we would be able to mock the `paymentWebhook` from the testing runtime and selectively have it perform its actual implementation or the mocked behavior, [like jest would let you d](https://jestjs.io/docs/mock-functions#mocking-partials)o.

Reply:
If you look at the PR from yesterday, I did that.  So, we have examples of both.